packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -280,7 +282,9 @@ if you want to create a different configuration of a UI controller.
       opened. It will force a content of an iframe to be reloaded and thus will solve display issues
       with a treebank view.
        */
-      experimentalResetTreebankURL: false
+      experimentalResetTreebankURL: false,
+      // A URL of a server that provides an app configuration
+      configServiceUrl: 'https://4wvouc833c.execute-api.us-east-2.amazonaws.com/prod/config'
```

Comment:
this isn't the final url, and probably the url shouldn't be in the code anyway or it should be able to be overridden, maybe via a create argument to the uicontroller. what do you think?

Reply:
or maybe for now we put it in a default feature options setting?

Reply:
> this isn't the final url, and probably the url shouldn't be in the code anyway or it should be able to be overridden, maybe via a create argument to the uicontroller. what do you think?

I've added this as an argument of a UI controller, and a calling app (embed lib or webextension) can override it. Let me submit a PR of a companion `embed-lib` changes that I've used for testing to demonstrate that.

If it's not a final URL then maybe we should not use it as a default value here and use `null` instead of relying on `embed-lib` or `webextension` to set the value? What do you think?

Reply:
ah, okay, great. As for default value, let's use https://config.alpheios.net/v1/config 

I will make the necessary terraform config today so that that url will answer.

Reply:
> As for default value, let's use https://config.alpheios.net/v1/config

Great! Updated a default to that value